### PR TITLE
Use formatDuration in LinkContents

### LINKED ISSUE #2996: Descriptive durations should just have raw numbers with a space
**Is your feature request related to a problem? Please describe.**
The problem with adding translations for minutes and seconds is that it is difficult to do for all languages, especially for some Slavic languages that have multiple versions of plurals. 

**Describe the solution you'd like**
Instead of the minutes/seconds in story promo headline, we should just have numbers with a space

1. Use formatDuration instead of describeDuration in LinkContents. 
2. Replace the colons between the hours/minutes/seconds with spaces in the formatDuration function. For example "1:00" -> "1 00". This will avoid screen reader interpreting the numbers as a time, eg 'one o'clock'. 

We can't think of a downside of having a space instead of a colon as the screenreader reads it in the same way when the numbers are not '1:00'. 

**Describe alternatives you've considered**
Loads - but this is the most straightforward thing to do. 

**Testing notes**
To be tested with all supported assistive technology. 

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by ghost**
Resolves #2996, #2309

**Overall change:** Format durations in `LinkContents` to fix a11y bug

**Code changes:**

- Add second arg `separator` to `formatDuration`.
- Use formatDuration in `LinkContents` with a empty space separator

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/StoryPromo/LinkContents/index.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import VisuallyHiddenText from '@bbc/psammead-visually-hidden-text';
 import pathOr from 'ramda/src/pathOr';
 import pick from 'ramda/src/pick';
 import { ServiceContext } from '../../../contexts/ServiceContext';
-import describeDuration from '../../../lib/utilities/describeDuration';
```
Comment (dr3) [inline src/app/containers/StoryPromo/LinkContents/index.jsx:8]: `describeDuration` is now not used by anything, can we delete it please

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/lib/utilities/formatDuration/index.js [hunk 2]
```diff
@@ -3,16 +3,16 @@
 const lpad = s => s.toString().padStart(2, '0');
 
 // the duration argument should be a moment duration!
-const formatDuration = duration => {
+const formatDuration = (duration, seperator = ':') => {
```
Comment (DenisHdz) [inline src/app/lib/utilities/formatDuration/index.js]:
```suggestion
const formatDuration = (duration, separator = ':') => {
```

File: src/app/lib/utilities/formatDuration/index.js [hunk 3]
```diff
@@ -3,16 +3,16 @@
...
   const isOverAnHour = duration.asHours() >= 1;
 
   let out = '';
 
   if (isOverAnHour) {
-    out += Math.floor(duration.asHours()) + ':';
-    out += lpad(duration.minutes()) + ':';
+    out += Math.floor(duration.asHours()) + seperator;
```
Comment (DenisHdz) [inline src/app/lib/utilities/formatDuration/index.js]:
```suggestion
    out += Math.floor(duration.asHours()) + separator;
```

File: src/app/lib/utilities/formatDuration/index.js [hunk 4]
```diff
@@ -3,16 +3,16 @@
...
+    out += lpad(duration.minutes()) + seperator;
```
Comment (DenisHdz) [inline src/app/lib/utilities/formatDuration/index.js]:
```suggestion
    out += lpad(duration.minutes()) + separator;
```

File: src/app/lib/utilities/formatDuration/index.js [hunk 5]
```diff
@@ -3,16 +3,16 @@
...
   } else {
-    out += duration.minutes() + ':';
+    out += duration.minutes() + seperator;
```
Comment (DenisHdz) [inline src/app/lib/utilities/formatDuration/index.js]:
```suggestion
    out += duration.minutes() + separator;
```

File: src/app/lib/utilities/formatDuration/index.test.js [hunk 6]
```diff
@@ -66,4 +66,12 @@ describe('formatDuration', () => {
     });
     expect(formatDuration(dur)).toBe('1:00:00');
   });
+
+  it('should apply the correct seperator', () => {
```
Comment (DenisHdz) [inline src/app/lib/utilities/formatDuration/index.test.js]:
```suggestion
  it('should apply the correct separator', () => {
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]
Apart from the typos this lgtm.

**Review by DenisHdz — APPROVED** [review]
Good one 👍

**Review by j-pendlebury — APPROVED** [review]
Looks good to me 👍

**Review by dr3 — APPROVED** [review]

**Comment by ghost** [comment]
Close in favour of #3194

**Pull request closed without merging**